chromium_src/chrome/browser/ui/webui/help/version_updater_mac.mm

```diff
@@ -32,14 +27,14 @@
 #endif
 
 // KeystoneObserver is a simple notification observer for Keystone status
-// updates. It will be created and managed by VersionUpdaterMac.
+// updates. It will be created and managed by SparkleVersionUpdater.
 @interface KeystoneObserver : NSObject {
  @private
-  raw_ptr<VersionUpdaterMac> versionUpdater_;  // Weak.
+  raw_ptr<SparkleVersionUpdater> versionUpdater_;  // Weak.
```

Comment:
are we renaming this `Sparkle` to differentiate old (sparkle) vs new (omaha 4)?

Reply:
Yes, exactly.